queries.sql

```diff
@@ -1,3 +1,43 @@
-/*Queries that provide answers to the questions from all projects.*/
-
 SELECT * from animals WHERE name = 'Luna';
+
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+--  List the name of all animals born between 2016 and 2019.
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' And '2019-01-01';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT name FROM animals WHERE neutered =TRUE And escape_attempts < 3;
+
+-- List date of birth of all animals named either "Agumon" or "Pikachu"
+SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
+
+-- List name and escape attempts of animals that weigh more than 10.5kg
+SELECT name,escape_attempts from animals WHERE weight_kg > 10.5;
+
+-- Find all animals that are neutered.
+SELECT * from animals WHERE neutered = TRUE;
+
+-- Find all animals not named Gabumon.
+SELECT * from animals WHERE name <> 'Gabumon';
+
+-- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
+SELECT * from animals WHERE weight_kg >=10.4 AND weight_kg <=17.3;
+
+-- How many animals are there?
+SELECT count(*) from animals;
+
+-- How many animals have never tried to escape?
+SELECT count(*) from animals where escape_attempts = 0;
+
+-- What is the average weight of animals?
+SELECT avg(weight_kg) from animals;
+
+-- Who escapes the most, neutered or not neutered animals?
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+
+-- What is the minimum and maximum weight of each type of animal?
+SELECT species,min(weight_kg), max(weight_kg) FROM animals GROUP BY species;
+
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+SELECT species,AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' And '2000-01-01' GROUP BY species;
```

Comment:
- [ ] Ending your check on the last day and last month of the year 2000 will ensure that all animals born inside the year 2000 will be fetched. 

See example:
```sql
SELECT species,AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' And '2000-31-12' GROUP BY species;
```

Reply:
I have made some changes can you please kindly check if I got what you were asking me to do?

Reply:
I have seen your changes @zairdon20. But it's still not right.  You need to end your check at `2000-31-12`
Please update your query to match this example: 
```sql
SELECT species,AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' And '2000-31-12' GROUP BY species;
```

Reply:
I also suggested something above. Please take a look at [this comment](https://github.com/zairdon20/Vet-clinic-database/pull/2#discussion_r839564809).

Reply:
This is what you mean, I have changed ?

Reply:
Yeah. Good job implementing them.👏